.circleci/config.yml

```diff
@@ -87,7 +87,11 @@ jobs:
           name: MyPy
           command: mypy sematic
       - run:
-          name: Run tests
+          name: Run Non-coverage Tests
+          # This assumes pytest is installed via the install-package step above
+          command: bazel test //sematic/... --test_output=all
+      - run:
+          name: Run Coverage Tests
```

Comment:
Why do we need both? I thought the `_coverage` targets also ran the actual tests.

Reply:
No, because of our bazelrc filter, only `"nocov"` tests get executed during a normal `bazel test`. You have to add `--test_tag_filters=cov` to get them.